configure.ac

```diff
@@ -497,6 +497,7 @@ AC_CHECK_DECL([SO_PASSCRED],
 
 AC_CHECK_FUNCS([ \
 	clearenv \
+	close_range \
 	__fpurge \
 	fpurge \
 	__fpending \
```

Comment:
You also need `UL_CHECK_SYSCALL([close_range])`

Reply:
I think f0649c0d83f30c7e2318ba0f578a55d875cca4d7 is now done as proposed.